.github/workflows/ci.yaml

```diff
@@ -70,6 +87,11 @@ jobs:
             composer-${{ runner.os }}-
             composer-
 
+      - name: Configure Symfony Flex
+        run: |
+            composer config extra.symfony.require ${{ matrix.execute-flex-with-symfony-version }}
+        if: "matrix.execute-flex-with-symfony-version"
```

Comment:
What do you think if `if:` would go before `run:`?

Reply:
No preference